--- org.freedesktop.upower.policy.orig	2016-09-24 17:32:43.000000000 +0000
+++ org.freedesktop.upower.policy	2016-11-05 21:28:58.903866602 +0000
@@ -21,6 +21,7 @@
     <defaults>
       <allow_inactive>no</allow_inactive>
       <allow_active>yes</allow_active>
+      <allow_any>yes</allow_any>
     </defaults>
   </action>
 
@@ -38,7 +39,8 @@
     <defaults>
       <allow_inactive>no</allow_inactive>
       <allow_active>yes</allow_active>
+      <allow_any>yes</allow_any>
     </defaults>
   </action>
 
